src/lib/Preview.js

```diff
@@ -982,6 +982,9 @@ class Preview extends EventEmitter {
         // Add the response interceptor to the preview instance
         this.options.responseInterceptor = options.responseInterceptor;
 
+        // Option to enable use of annotations v4
+        this.options.enableAnnotations = options.enableAnnotations || false;
```

Comment:
What do you think about naming this `showAnnotationsControls` and grouping it with `showAnnotations`? It can also be simplified to `this.options.showAnnotationsControls = !!options.showAnnotationsControls`;